src/components/BookmarkCard/index.tsx

```diff
@@ -28,9 +31,26 @@ const BookmarkCard = ({
   acronym,
   status,
   statusIcon,
-  businessOwnerOrg
+  businessOwnerOrg,
+  refetch
 }: BookmarkCardProps & CedarSystemProps) => {
   const { t } = useTranslation();
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [deleteMutate, mutationResult2] = useMutation(
```

Comment:
You can just use `const [deleteMutate] = useMutation(...)` here to ignore the mutation's result; this also lets you get rid of the eslint-disable-next-line.